components/list/list.js

```diff
@@ -368,6 +371,8 @@ class List extends PageableMixin(SelectionMixin(LitElement)) {
 
 	_handleSlotChange() {
 		this._updateItemShowingCount();
+		const items = this.getItems();
+		if (items.length > 0) items[0].first = true;
```

Comment:
Do we need to iterate over all the items to ensure that all other items have `first = false`?  For example, are there scenarios where a new item could be added to the top of the list (by whatever means, possibly drag & drop or an action performed in the list-controls header, etc), such that the initial item with `first = true` becomes the second item and should be updated as such?

Reply:
Good call!